ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -70,11 +77,11 @@ const AdvancedGasFeeDefaults = () => {
           disabled={hasErrors}
         />
         <Typography variant={TYPOGRAPHY.H7} color={COLORS.UI4} margin={0}>
-          {!isDefaultSettingsSelected && Boolean(advancedGasFeeValues)
-            ? t('advancedGasFeeDefaultOptIn', [
+          {isDefaultSettingsSelected
```

Comment:
double-checking, would it make sense to use `isDefaultSettingsSelected || !Boolean(advancedGasFeeValues)` here?

Reply:
`isDefaultSettingsSelected` makes UI optimistic and avoid lag resulting from state change.